src/xa/xml/maybes.js

```diff
@@ -1,8 +1,17 @@
 import * as _ from 'lodash';
 
 function maybe_find_one(pn, xp, attrs = [], fn = null) {
```

Comment:
Both changes in this function are equivalent. Factor them into a single function.